frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/issue-template.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CdkDragDrop, moveItemInArray, transferArrayItem } from '@angular/cdk/drag-drop';
+import { TasqueDropdownOption } from 'src/shared/components/tasque-dropdown/dropdown.component';
+import { TaskTemplate } from 'src/core/models/task/task-template';
+import { ToastrService } from 'ngx-toastr';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { TaskCustomField } from 'src/core/models/task/task-custom-field';
+import { TaskFieldType } from 'src/core/models/task/task-field-types';
+import { Router } from '@angular/router';
+import { AvailableFields } from 'src/core/models/const-resources/available-fields';
+
+@Component({
+  selector: 'app-issue-template',
+  templateUrl: './issue-template.component.html',
+  styleUrls: ['./issue-template.component.sass']
+})
+export class IssueTemplateComponent implements OnInit {
+
+  constructor(
+    private notificationService: ToastrService,
+    private router: Router,
+  ) { }
+
+  @Input() public issueTemplate: TaskTemplate;  
+  //fieldsWithDescription and contextFields props should be replaced with this object same properties
+
+  public fieldsWithDescription: TaskCustomField[] = [ 
+    { name: 'Description', type: TaskFieldType.Paragraph }, 
+    { name: 'Summary', type: TaskFieldType.Text }, 
+    { name: 'State', type: TaskFieldType.State }, 
+    { name: 'Type', type: TaskFieldType.Type }
+  ];
+  public contextFields: TaskCustomField[] = [ 
+    { name:'Assignee', type: TaskFieldType.User }, 
+    { name:'Label', type: TaskFieldType.Label, labels: [ 
+      { color: 'red', name: 'first label' },
+      { color: 'green', name: 'second label' },
+      { color: 'blue', name: 'third label' }, 
+    ] }, 
+    { name:'Sprint', type: TaskFieldType.Type }, 
+    { name:'Story point estimate', type: TaskFieldType.Number }
+  ];
```

Comment:
I hope you will remove this custom data

Reply:
Yes, this fields will be taken from existing task templates by fetching data. This was only for test how implemented functionality works. At this time I need to use this data with functionality in branch with noSql to find a better way to connect both features together.